src/components/Alert.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Alert = ({ alert }) => {
+  return <div className={`alert alert-${alert.type}`}>{alert.text}</div>;
+};
+
+export default Alert;
```

Comment:
Its good enough for now but I would rather use material ui Snackbar.
https://material-ui.com/components/snackbars/

When using material ui should focus styles to be included in javascript instead of css.